nginx/server_name

```diff
@@ -0,0 +1 @@
+server_name ${DOMAIN} www.${DOMAIN};
```

Comment:
Looking at the existing instances at [joinbookwyrm.com](https://joinbookwyrm.com/instances) it seems about 50% are already using a subdomain. I therefore think automatically including `www.${DOMAIN}` is unnecessary and creates manual work for a lot of admins since they're unlikely to want `www.subdomain.example.com` – i.e. I think if they _do_ want it they should add it, rather than making it the default to include a `www` subdomain or sub-subdomain.